guides/micronaut-docker-image/micronaut-docker-image.adoc

```diff
@@ -124,7 +124,7 @@ Dockerfile written to: /Users/sdelamo/github/micronaut-projects/micronaut-guides
 
 > Task :dockerBuildNative
 ...
-Step 1/14 : FROM ghcr.io/graalvm/native-image:ol7-java17-22.3.0 AS graalvm
+Step 1/14 : FROM ghcr.io/graalvm/native-image-community:17 AS graalvm
```

Comment:
Nit: Shouldn't this be

```suggestion
Step 1/14 : FROM ghcr.io/graalvm/native-image-community:17-ol9 AS graalvm
```

(see https://github.com/micronaut-projects/micronaut-gradle-plugin/blob/4a10d81a310d405c2d42851b382c003dd5a4484c/docker-plugin/src/main/java/io/micronaut/gradle/docker/NativeImageDockerfile.java#L382)